04 Strategy Library/118 Time Series Momentum Effect/02 Method.html

```diff
@@ -36,7 +35,7 @@
   In <code>OnData(self, data)</code>, indicators for all futures contracts are updated every day with the settlement price.
 </p>
 <p>
-  We choose instruments with positive 12-month momentum to long, and instrucments with negative 12-month momentum to short.
+  We take a long position in instruments with positive 12-month momentum and short positions with negative momentum.
```

Comment:
Change to:

`We take a long position in instruments with positive 12-month momentum and short positions in those with negative momentum.`